app/controllers/ctc/portal/refund_payment_controller.rb

```diff
@@ -0,0 +1,32 @@
+class Ctc::Portal::RefundPaymentController < Ctc::Portal::BaseIntakeRevisionController
+
+  def edit
+    @form = form_class.new(current_intake)
+    render edit_template
+  end
+
+  private
+
+  def next_path
+    if current_intake.refund_payment_method_direct_deposit?
+      redirect_to ctc_portal_bank_account_path
+    elsif current_intake.refund_payment_method_check?
+      redirect_to ctc_portal_mailing_address_path
+    else
+      redirect_to Ctc::Portal::PortalController.to_path_helper(action: :edit_info)
+    end
+  end
+
+  def edit_template
+    "ctc/portal/refund_payment/edit"
+  end
+
+  def form_class
+    Ctc::Portal::RefundPaymentForm
+  end
```

Comment:
test to see what happens if we don't define these? will it find the view and the form?

Reply:
update: it doesn't work